lib/api/request/requestContext.ts

```diff
@@ -20,7 +20,7 @@
  */
 
 import * as assert from '../../util/assertType';
-import { JSONObject } from '../../util/interfaces';
+import { JSONObject } from 'kuzzle-sdk';
```

Comment:
Ok now I see why we need a `@types/kuzzle` package...

Reply:
I need some more time to understand everything we need in this `@types/kuzzle` package and then I will create it.